CHANGELOG.md

```diff
@@ -27,6 +27,8 @@ _This release is scheduled to be released on 2023-04-01._
 
 - Use develop as target branch for dependabot
 - Update issue template and contributing doc
+- The weather modules clearly separates precipation amount and probability (risk of rain/snow)
+  - This requires all providers that only supports probability to change the config from `showPrecipitationAmount` to `showPrecipitation` or `showPrecipitationProbability`.
```

Comment:
Those config options need to be added to the default config: https://github.com/MichMich/MagicMirror/blob/42d42ef4526be1f74ef9570a81e7027592e072f5/modules/default/weather/weather.js#L11